keyworker-specs/src/test/groovy/uk/gov/justice/digital/hmpps/keyworker/specs/KeyworkerPrisonStatsSpecification.groovy

```diff
@@ -167,6 +168,7 @@ class KeyworkerPrisonStatsSpecification extends GebReportingSpec{
 
         header.dropDown.click()
         header.brixtonCaseLoad.click()
+        elite2api.stubGetMyCaseloads(ITAG_USER.caseloads, BXI.id)
```

Comment:
This is to fix a test that broke when caseload active indicator work was done, not sure how it passed before...